### PR TITLE
some cleanup to release checklist

### PR DESCRIPTION
**Pull request by jessfraz**
some more cleanup to release checklist

### CODE REVIEW

File: project/RELEASE-CHECKLIST.md [hunk 1]
```diff
@@ -222,42 +226,42 @@ another docker/docker directory in bundles. This next step assumes you have
 a checkout of the docker source code at the same commit you used to build, with
 the artifacts from the last step in `bundles`.
 
+**NOTE:** If you put a space before the command your `.bash_history` will not
+save it. (for the `GPG_PASSPHRASE`).
```
Comment (thaJeztah) [inline project/RELEASE-CHECKLIST.md]: Never actually knew that!

**Comment by thaJeztah** [comment]
LGTM

ping @tiborvass

**Comment by tiborvass** [comment]
LGTM

File: project/RELEASE-CHECKLIST.md (hunk 1 above)
Comment (albers) [inline project/RELEASE-CHECKLIST.md]: Wow, that's kinda cool.

**Pull request merged**